fcomm/examples/Makefile

```diff
@@ -16,29 +16,41 @@ fibonacci-proof2.json : fibonacci-claim.json
 fibonacci-proof : fibonacci-proof.json
 fibonacci-proof2 : fibonacci-proof2.json
 
-verify-fibonacci-proof :
+verify-fibonacci-proof : fibonacci-proof.json
 	cargo run --release -- verify --proof fibonacci-proof.json
 
-num-list-commitment.json : num-list-function.json
+verify-fibonacci-proof2 : fibonacci-proof2.json
+	cargo run --release -- verify --proof fibonacci-proof2.json
+
+num-list-commitment : num-list-function.json
 	cargo run --release -- commit --function num-list-function.json --commitment num-list-commitment.json
 
-num-list-input.lurk : num-list-input.lurk
+num-list-opening : num-list-opening.json
 
-num-list-opening.json : num-list-function.json num-list-input.lurk
+num-list-opening.json : num-list-function.json num-list-input.lurk num-list-commitment
 	cargo run --release -- open --function num-list-function.json --input num-list-input.lurk --proof num-list-opening.json
 
-num-list-proof : num-list-opening.json
-
-verify-num-list-opening :
+verify-num-list-opening : num-list-opening.json
 	cargo run --release -- verify --proof num-list-opening.json
 
-chained-commitment.json : chained-function.json
+chained-commitment : chained-function.json
 	cargo run --release -- commit --function chained-function.json --commitment chained-commitment.json
 
 chained-opening: chained-opening.json
 
-chained-opening.json : chained-function.json chained-input.lurk
+chained-opening.json : chained-function.json chained-input.lurk chained-commitment.json
 	cargo run --release open --function chained-function.json --input chained-input.lurk --proof chained-opening.json --chain
 
-verify-chained-opening :
+verify-chained-opening : chained-opening.json
 	cargo run --release -- verify --proof chained-opening.json
+
+chained-opening2: chained-opening2.json
+
+chained-opening2.json : chained-opening.json
+	# NOTE: This concrete commitment (39ceb60c198b6b7c9f1ff98dd83026bd282e32307d8bd10a28f19d637c4d8549) can be found
+	# in chained-opening2.json, but its identity is determistic based on the secret and 'second return value' (cdr)
+	# of the function specified in chained-function.json.
+	cargo run --release open --commitment 39ceb60c198b6b7c9f1ff98dd83026bd282e32307d8bd10a28f19d637c4d8549 --input chained-input.lurk --proof chained-opening2.json --chain
```

Comment:
I’m not sure I understand this comment? Is it also the right place to comment, given `chained-opening2.json` is the target, rather than the dependency, of this rule?

Reply:
Whoops, that should read `chained-opening.json`, which should address your concern. Good catch, thanks.

The point of the comment is that this commitment value is something of a magic number. If in the future, the commitment mechanism, the input, or the chained function itself change, then this 'next commitment' will also change. The comment alerts a future developer how to find the right commitment to use in updating this rule.

You could also do it by manually running the function on the input and observing the output, but since that is already done and reported in `chained-opening.json`, fetching it from there might be simpler.